curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/63ff5850cd098f1796b5e9fa.md

```diff
@@ -0,0 +1,136 @@
+---
+id: 63ff5850cd098f1796b5e9fa
+title: Step 3
+challengeType: 0
+dashedName: step-3
+---
+
+# --description--
+
+When you make a `GET` request to an endpoint, the data you get is not always usable at first. To make it usable, you can use the `json()` method on the response with the `.then()` method.
+
+The `.then()` method can take a callback function for the fulfilled and rejected cases of the `Promise`.
+
+Add a `.then()` method right under your `fetch()`. Inside the `.then()` method, name the response from the endpoint `res`, add a fat arrow for the callback, then use the `json()` method on that `res`. You don't need to use curly braces because the arrow function will have one expression. 
```

Comment:
I think we should the work chain, to help reinforce that concept to campers. Maybe something like this. 

```suggestion
Chain the `.then()` method to your fetch call. Inside the `.then()` method, add a callback function with a `res` parameter that returns `res.json()`. You don't need to use curly braces because the arrow function will have one expression. 
```